spec/models/requests/requestable_decorator_spec.rb

```diff
@@ -1428,7 +1436,7 @@
   end
 
   describe "#off_site?" do
-    let(:stubbed_questions) { default_stubbed_questions.merge(etas?: false, item_data?: false, circulates?: true) }
+    let(:stubbed_questions) { default_stubbed_questions.merge(etas?: false, item_data?: false, recap_pf?: false, circulates?: true) }
```

Comment:
recap_pf?: true

Reply:
I doubt it matters,  Off Site is the location it is being held and has nothing to do with delivery (or should not).

Reply:
It is odd that recap_pf? is needed here `recap? || recap_pf? || annex? || item_at_clancy? || held_at_marquand_library?` will evaluate to true if recap? is true.